common/Util.hpp

```diff
@@ -604,19 +604,16 @@ namespace Util
 
     size_t findInVector(const std::vector<char>& tokens, const char *cstring, std::size_t offset = 0);
 
-    inline std::string& trim(std::string& s, const char ch)
+    /// Trim trailing characters (on the right).
+    inline std::string_view trim(const std::string_view s, const char ch)
```

Comment:
After this, we now have two trims. This one where with a 2nd arg which doesn't modify the 1st arg, while the other one with no 2nd arg does modify its string arg.

Checking, it looks like all current uses of this 2 arg trim already assign the return to what was passed as 1st arg (two in common/JailUtil.cpp) so no change in practice, but the discrepancy is annoying

Reply:
Good point. Let me check and eliminate.